src/components/App.jsx

```diff
@@ -0,0 +1,24 @@
+
+import { Routes, Route } from "react-router-dom"
+import { useState } from "react"
+import Header from './Header'
+import Home from './Home'
+import Articles from './Articles'
+import '../App.css'
+
+function App() {
+
+  const [articles, setArticles] = useState([])
```

Comment:
consider where you'd like to declare the articles states. If it's not going to used anywhere else, probably better to declare it in Articles component rather than passing it down as props